src/main/scala/com/mwz/sonar/scala/scoverage/ScoverageSensor.scala

```diff
@@ -162,7 +178,8 @@ private[scoverage] abstract class ScoverageSensorInternal extends Sensor {
 
 object ScoverageSensorInternal {
   private val SensorName = "Scoverage Sensor"
-  private val ScoverageReportPathPropertyKey = "sonar.scoverage.reportPath"
+  private val OldScoverageReportPathPropertyKey = "sonar.scoverage.reportPath"
+  private val NewScoverageReportPathPropertyKey = "sonar.scala.scoverage.reportPath"
```

Comment:
Personally, I'd name the new key just `ScoverageReportPathPropertyKey` and the deprecated one `DeprecatedScoverageReportPathPropertyKey` but I'm not too fussy about this.

Reply:
ok